content/2. Ondersteunende-informatie/3. Processen/4. Implementatieproces/1. Inrichten ontwikkelomgeving/1. IDE setup en build.md

```diff
@@ -0,0 +1,60 @@
+---
+title:
+  - IDE setup en build
+tags:
+---
+
+## Wat is een IDE?
+Een integrated development environment of IDE is een programma die een softwareontwikkelaar ondersteunt bij het ontwikkelen van software. Het bestaat uit een aantal onderdelen, zoals de mogelijkheid om een gebouwd programma te runnen en te controleren op fouten (debuggen). 
+Voor C# gebruik je als IDE de laatste versie van [Visual Studio Community Edition](https://visualstudio.microsoft.com/downloads/)
+
+## Hoe zit de IDE in elkaar?
+De IDE bevat alles wat je nodig hebt om code te kunnen schrijven en onderhouden. Voor de C# onderdelen wordt Visual Studio gebruikt als IDE. In de afbeelding hieronder zie je wat Visual Studio te bieden heeft:
+![[Functies IDE.png]]
+
+### Installeren van de IDE Visual Studio?
+Je vindt de laatste versie van Visual Studio bij Microsoft downloads. Let op we gebruiken de Community Edition: [Visual Studio Community Edition](https://visualstudio.microsoft.com/downloads/).
+Na downloaden kun je het product installeren met de default instellingen. Later kun je dit met de Visual Studio Installer uitbreiden of wijzigen als dat nodig is. 
+## Hoe gebruik je de IDE?
+Om de IDE te leren kennen werk je vanuit een eenvoudige casus:
+
+> [!TIP] Casus
+>[Schrijf een Hello World applicatie. De applicatie print in een console scherm de tekst 'Hello World']
+
+**Uitwerking van de casus**
+```C#
+Console.WriteLine("Hello, World!");
+```
+### Stappenplan
+1. Start Visual Studio.
+2. Kies: Create New Project.
+3. Kies: C# Console App en klik op Next.
+4. Kies: Naam, b.v. HelloWorldApp en locatie waar het project bewaard mag worden.
+5. Kies als framework voor .NET8 (Long term support) en klik op Create.
+6. Visual Studio wordt opgestart en is klaar voor gebruik als de class Program met de print opdracht voor Hello World in het beeld verschijnt.
+7. Kies in het hoofdmenu voor de optie **Build** en vervolgens **Build Solution** om het project te compileren. Als er geen fouten optreden is het programma klaar om gestart te worden. In het menu zie je ook dat standaard **F6** de sneltoets voor Build is. Dus je kunt **F6** gebruiken om  meteen Build te starten.
+8. Kies uit het hoofdmenu voor **Debug** en vervolgens **Start Without Debugging** om het programma te runnen. Dit kan ook rechtstreeks met de toetscombinatie **CTRL**+**F5**.
+
+### Oefening
+Probeer nu zelf de volgende oefening uit te voeren in een nieuw project door de bovenstaande acht stappen uit te voeren.
+
+> [!TIP] Casus
+>[Schrijf de getallen 1 t/m 10 onder elkaar op een console scherm]
+
+
+**Uitwerking van de casus**
+```C#
+for(int i = 1; i < 11; i++)
+{
+    Console.WriteLine(i);
+}
+```
+
+### Debug/Release mode
+
+Standaard staat het project in debug mode. Dit betekent dat er extra informatie wordt toegevoegd
```

Comment:
Hier kan heel mooi een verwijzing gemaakt worden naar het onderwerp debugging (Deze is nog niet uitgewerkt, maar zal later erbij gevoegd kunnen worden)